pkg/webhook/admission/validator.go

```diff
@@ -29,9 +30,9 @@ import (
 // Validator defines functions for validating an operation.
 type Validator interface {
 	runtime.Object
-	ValidateCreate() error
-	ValidateUpdate(old runtime.Object) error
-	ValidateDelete() error
+	ValidateCreate() ([]string, error)
```

Comment:
We'd better add comments for these changed method. `[]string` is a confusing return value and users have to read the source code to see what is the `[]string` for.

Reply:
Updated! PTAL @FillZpp